src/utilities/utilities.ts

```diff
@@ -13,12 +16,41 @@ export const cleanList = (restaurants: RestaurantsState[]) => {
       alt: restaurant.attributes['alt_text'],
       location: {
         lat: restaurant.attributes.lat,
-        long: restaurant.attributes.long
+        long: restaurant.attributes.long,
       },
       alcoholic: restaurant.attributes['alcholic_drinks'], // fix spelling
       happyHour: restaurant.attributes['happy_hour'],
-      hours: restaurant.relationships[0].hours,
-      happyHours: restaurant.relationships[0]['happy_hours']
-    }
+      hours: cleanedHours,
+      happyHours: cleanedHappyHours,
+    } as CleanedRestaurantsState
+  })
+}
+
+const cleanHours = (restaurant: RestaurantsState) => {
```

Comment:
Makes sense to clean the data within hours and happy hours to get rid of those snake case properties. Nice job catching this!